boards.txt

```diff
@@ -4774,3 +4774,45 @@ metro_esp-32.menu.DebugLevel.verbose=Verbose
 metro_esp-32.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
+sensesiot_weizen.name=Senses's WEIZEN
+
+sensesiot_weizen.upload.tool=esptool_py
+sensesiot_weizen.upload.maximum_size=1310720
+sensesiot_weizen.upload.maximum_data_size=327680
+sensesiot_weizen.upload.wait_for_upload_port=true
+
+sensesiot_weizen.serial.disableDTR=true
+sensesiot_weizen.serial.disableRTS=true
+
+sensesiot_weizen.build.mcu=esp32
+sensesiot_weizen.build.core=esp32
+sensesiot_weizen.build.variant=sensesiot_weizen
```

Comment:
you did not add a header for the variant. If this is same as ESP32 dev boards, change the variant to "esp32", else you need to add the "pins_arduino.h" in the proper variant subfolder

Reply:
OK, I see.

Reply:
I updated it already, please verify.
Thank you.